### PR TITLE
Issue #91

### PR DESCRIPTION
**Pull request by jroden**
Report number of unique values for categorical variables

### CODE REVIEW

File: src/codebook/summaryTable/renderRow/makeDetails.js [hunk 1]
```diff
@@ -25,6 +25,17 @@ export default function makeDetails(d) {
       .attr("class", "stat");
     stats.append("div").text(d => d.key).attr("class", "label");
     stats.append("div").text(d => d.value).attr("class", "value");
+	// Display Unique Values for Categorical Variables
```
Comment (jwildfire) [inline src/codebook/summaryTable/renderRow/makeDetails.js]: This works, but there's an easier way.  Just add the number of distinct levels to the summary data object for categorical variables in [this method](d3.set](https://github.com/RhoInc/web-codebook/blob/181bf34cd1715ec8078a3f5dfaaba1c7332fa2c2/src/codebook/data/makeSummary.js) using [d3.set](https://github.com/RhoInc/web-codebook/blob/181bf34cd1715ec8078a3f5dfaaba1c7332fa2c2/src/codebook/data/makeSummary.js#L17)

Once that variable is added to the data object, the new 'unique' statistic should renderer automatically without any updates to this file.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1867 @@
+{
```
Comment (jwildfire) [inline package-lock.json]: you can delete this file

File: src/codebook/data/makeSummary.js [hunk 3]
```diff
@@ -37,7 +37,8 @@ export function makeSummary(codebook) {
             prop_N: d.length / statistics.N,
             prop_n: d.length / statistics.n,
             prop_N_text: d3format("0.1%")(d.length / statistics.N),
-            prop_n_text: d3format("0.1%")(d.length / statistics.n)
+            prop_n_text: d3format("0.1%")(d.length / statistics.n),
+		  unique: d3set(statistics.values.length - 5)
```
Comment (jwildfire) [inline src/codebook/data/makeSummary.js]: Looks good! You can delete the  "-5"

**Comment by jwildfire** [comment]
Also make sure to run `npm run build` and commit the updated compiled code.

**Comment by jroden** [comment]
on it

**Comment by jroden** [comment]
should be g2g

**Comment by jwildfire** [comment]
Had lots of conflicts, so just copying this to a fresh branch/PR - see #120

**Pull request closed without merging**